spec/controllers/admin/admin_custom_files_controller_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Admin::CustomFilesController, type: :controller do
+  describe 'Slides' do
+    let(:admin_user) { create(:admin_user) }
+    before do
+      login_by_admin_user admin_user
+    end
+
+    describe 'GET /admin/custom_files/' do
+      it 'works!' do
+        get 'index'
+        expect(response.status).to eq(200)
+      end
+    end
+
+    describe 'GET /admin/custom_files/new' do
+      it 'works!' do
+        get 'new'
+        expect(response.status).to eq(200)
+      end
+    end
+
+    describe 'POST /admin/custom_files' do
+      it 'works!' do
+        allow(SlideHub::Cloud::Engine::AWS).to receive(:upload_files).and_return(true)
+        allow(SlideHub::Cloud::Engine::Azure).to receive(:upload_files).and_return(true)
+        post 'create', params: { custom_file: {file: fixture_file_upload('dummy.txt', "text/html"), description: 'hoge'} }
+        expect(response.status).to eq(302)
+        expect(response).to redirect_to admin_custom_files_path
+        expect(File.exist?(Rails.root.join('tmp', 'dummy.txt'))).to eq(false)
+      end
+
+      it 'updates the existing record if path is equal to the existing one' do
+        allow(SlideHub::Cloud::Engine::AWS).to receive(:upload_files).and_return(true)
+        allow(SlideHub::Cloud::Engine::Azure).to receive(:upload_files).and_return(true)
+        custom_file = create(:default_custom_file)
+        post 'create', params: { custom_file: {file: fixture_file_upload('dummy.txt', "text/html"), description: 'hoge'} }
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->